README.md

```diff
@@ -2,39 +2,67 @@
 </p>
+
 <hr>
 
 <p align="center">
-  <a href="https://docs.zama.ai/concrete-ml"> 📒 Read documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
+  <a href="https://docs.zama.ai/concrete-ml"> 📒 Documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources by Zama</a>
 </p>
+
 <p align="center">
-<!-- Version badge using shields.io -->
-  <a href="https://github.com/zama-ai/concrete-ml/releases">
-    <img src="https://img.shields.io/github/v/release/zama-ai/concrete-ml?style=flat-square">
-  </a>
-<!-- Link to tutorials badge using shields.io -->
-  <a href="#license">
-    <img src="https://img.shields.io/badge/License-BSD--3--Clause--Clear-orange?style=flat-square">
-  </a>
-<!-- Zama Bounty Program -->
-  <a href="https://github.com/zama-ai/bounty-program">
-    <img src="https://img.shields.io/badge/Contribute-Zama%20Bounty%20Program-yellow?style=flat-square">
-  </a>
+  <a href="https://github.com/zama-ai/concrete-ml/releases"><img src="https://img.shields.io/github/v/release/zama-ai/concrete-ml?style=flat-square"></a>
+  <a href="license"><img src="https://img.shields.io/badge/License-BSD--3--Clause--Clear-%23ffb243?style=flat-square"></a>
+  <a href="https://github.com/zama-ai/bounty-program"><img src="https://img.shields.io/badge/Contribute-Zama%20Bounty%20Program-%23ffd208?style=flat-square"></a>
 </p>
-<hr>
 
-Concrete ML is a Privacy-Preserving Machine Learning (PPML) open-source set of tools built on top of [Concrete](https://github.com/zama-ai/concrete) by [Zama](https://github.com/zama-ai). It simplifies the use of fully homomorphic encryption (FHE) for data scientists to help them automatically turn machine learning models into their homomorphic equivalent. Concrete ML was designed with ease-of-use in mind, so that data scientists can use it without knowledge of cryptography. Notably, the Concrete ML model classes are similar to those in scikit-learn and it is also possible to convert PyTorch models to FHE.
+## About
+
+### What is Concrete ML
+
+**Concrete ML** is a Privacy-Preserving Machine Learning (PPML) open-source set of tools built on top of [Concrete](https://github.com/zama-ai/concrete) by [Zama](https://github.com/zama-ai).
+
+It simplifies the use of fully homomorphic encryption (FHE) for data scientists so that they can automatically turn machine learning models into their homomorphic equivalent, and use it without knowledge of cryptography.
+
+Concrete ML is designed with ease of use in mind. Data scientists can use models with APIs that are close to the frameworks they use, while additional options to those models allow them to run inference or training on encrypted data with FHE. Notably, the Concrete ML model classes are similar to those in scikit-learn and it is also possible to convert PyTorch models to FHE.
```

Comment:
suggest some rewriting: 
- `close to the frameworks they already know well`
- `Notably, the Concrete` -> remove `Notably`
- some commas needed

whole paragraph:

```
Concrete ML is designed with ease of use in mind. Data scientists 
can use models with APIs that are close to the frameworks they 
already know, and new options to those models allow inference 
or training on encrypted data with FHE. Concrete ML model classes 
are similar to those in scikit-learn and it is also possible to convert PyTorch models to FHE.
```

Reply:
✅